guides/md/serialization_and_saving.md

```diff
@@ -292,17 +310,26 @@ reconstructed_model.fit(test_input, test_target)
 
 <div class="k-default-codeblock">
 ```
-4/4 [==============================] - 0s 967us/step - loss: 0.8106
-4/4 [==============================] - 0s 1ms/step - loss: 0.7184
+4/4 [==============================] - 0s 2ms/step - loss: 0.2871
+4/4 [==============================] - 0s 2ms/step - loss: 0.2784
 
-<tensorflow.python.keras.callbacks.History at 0x151d5ac90>
+<keras.callbacks.History at 0x79fb1454b4a8>
 
 ```
 </div>
-#### Limitations
+### Format Limitations
+
+SavedModel limitations:
+- Can be slower and bulkier than H5, since it saves the traced TF graphs of each layer
+- Does not support layers with masks (`layer.supports_masking=True`)
```

Comment:
This isn't really accurate, is it? Most Keras layers support masking, e.g. Dense, LSTM, etc. If they weren't supported, virtually no model could be saved.

Reply:
I reworded this to mention this is the case for custom layers. There is also a note below saying that if the object as a get_config/from_config, then the masking will work.